crates/extensions/c8y_mapper_ext/src/alarm_converter.rs

```diff
@@ -36,9 +37,10 @@ impl AlarmConverter {
     pub(crate) fn try_convert_alarm(
         &mut self,
         source: &EntityTopicId,
+        external_id: &EntityExternalId,
+        entity_type: &EntityType,
         input_message: &MqttMessage,
         alarm_type: &str,
-        entity_store: &EntityStore,
```

Comment:
Oh, the *whole* entity_store was passed to convert a specific alarm !? Good to have that fixed.